server/swap/swap.go

```diff
@@ -1135,6 +1135,16 @@ func (s *Swapper) readMatches(matchSets []*order.MatchSet) []*matchTracker {
 	return matches
 }
 
+// extractAddress extracts the address from the order. If the order is a cancel
+// order, an empty string is returned.
+func extractAddress(ord order.Order) string {
+	trade := ord.Trade()
+	if trade == nil {
+		return ""
+	}
+	return trade.Address
+}
```

Comment:
This is functionally what `Order.FilledAmt` did, but my goal was to get rid of all methods that had special return values from `CancelOrder` (except the new `Trade` method), so that's my starting point.

Reply:
This is a good helper for that purpose.  The goal is to make the exported Order interface cleaner, and some internal helpers like this are A-OK imo.